pkg/downstream/downstream.go

```diff
@@ -58,3 +78,36 @@ func SetDownstreamVersionPendingPreflight(appID string, sequence int64) error {
 
 	return nil
 }
+
+// TODO: this needs cluster ID
+func ShouldIgnoreRBACErrors(appID string, sequence int64) (bool, error) {
+	db := persistence.MustGetPGSession()
+	query := `SELECT preflight_ignore_permissions FROM app_downstream_version
+	WHERE app_id = $1 and sequence = $2 LIMIT 1` // this need cluster ID to avoid returning multiple rows
+	row := db.QueryRow(query, appID, sequence)
+
+	var shouldIgnore sql.NullBool
+	if err := row.Scan(&shouldIgnore); err != nil {
+		return false, errors.Wrap(err, "failed to select downstream")
+	}
+
+	if !shouldIgnore.Valid {
+		return false, nil
+	}
+
+	return shouldIgnore.Bool, nil
+}
+
+func SetIgnorePreflightPermissionErrors(appID string, clusterID string, sequence int64) error {
```

Comment:
should this share the name of the above function? Like, have one be 'set' and the other 'get' or something